pkg/pxc/service.go

```diff
@@ -207,6 +215,13 @@ func NewServiceProxySQL(cr *api.PerconaXtraDBCluster) *corev1.Service {
 	return obj
 }
 
+func NewServiceProxySQLName(cr *api.PerconaXtraDBCluster) types.NamespacedName {
```

Comment:
Consider rename to ProxySQLServiceNamespacedName

Reply:
fbb29694